dev/fattest.simplicity/src/componenttest/topology/utils/HttpsRequest.java

```diff
@@ -182,6 +183,8 @@ public void checkServerTrusted(X509Certificate[] certs, String authType) {}
 
             if (json != null) {
                 con.setRequestProperty("Content-Type", "application/json");
+                //TODO Look for alternatives to output stream for writing JSON data to message body.
+                //This line will change request method to POST
```

Comment:
since we can break backwards compatibility in the test framework, we could save others the confusion and throw an `IllegalStateException` or something if they try to specify `json` and `method("GET")` at the same time